tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1,6 +1,414 @@
 /* BEGIN_HEADER */
 #include "test/drivers/test_driver.h"
 
+/* Auxiliary variables for pake tests.
+   Global to silent the compiler when unused. */
+size_t pake_expected_hit_count = 0;
+int pake_in_driver = 0;
+#if defined(PSA_WANT_ALG_JPAKE) && defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR) && \
+    defined(PSA_WANT_ECC_SECP_R1_256) && defined(PSA_WANT_ALG_SHA_256)
```

Comment:
`ecjpake_do_round` is a `static` auxiliary function used in `ecjpake_rounds` test function. I think that dependencies need to match for both as I'm getting unused function warning(e.g. build_psa_accel_alg_md5):
`suites/test_suite_psa_crypto_driver_wrappers.function:9:13: error: ‘ecjpake_do_round’ defined but not used [-Werror=unused-function]
`

Reply:
ok make sense, should we do the same for test_suite_psa_crypto_pake.function::ecjpake_do_round?

Reply:
There was no problem with this function on CI. I think because in `test_suite_psa_crypto_pake.function` `ecjpake_do_round` is guarded by `PSA_WANT_ALG_JPAKE` and `ecjpake_rounds` and `ecjpake_rounds_inject` test functions depend also only on `PSA_WANT_ALG_JPAKE`.